assets/models/system/databricks-dolly-v2-12b/spec.yaml

```diff
@@ -2,20 +2,29 @@ $schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
 name: databricks-dolly-v2-12b
 path: ./
 properties:
-  SHA: 7dde7ac26ddcb679e0429fed1b6f751c1ffab1fb
+  SHA: 19308160448536e378e3db21a73a751579ee7fdd
   datasets: databricks/databricks-dolly-15k
   evaluation-min-sku-spec: 16|0|64|128
   evaluation-recommended-sku: Standard_NC24s_v3
```

Comment:
lets update this as well with list of comma sep. SKUs

Reply:
Done